test/applied-filters/applied-filters.test.js

```diff
@@ -2,7 +2,7 @@ import '../../components/filter-dropdown/filter-dropdown.js';
 import '../../components/filter-dropdown/filter-dropdown-category.js';
 import '../../components/filter-dropdown/filter-dropdown-option.js';
 import '../../components/applied-filters/applied-filters.js';
-import { expect, fixture, html, oneEvent } from '@open-wc/testing';
+import { clickElem, expect, fixture, html, oneEvent, sendKeysElem, waitUntil } from '@brightspace-ui/testing';
 import { runConstructor } from '@brightspace-ui/core/tools/constructor-test-helper.js';
```

Comment:
Would you be able to convert the other 2 files using `@open-wc/testing` over as well? Looks like they were missed when this repo was migrated to `@brightspace-ui/testing`.

```suggestion
import { clickElem, expect, fixture, html, oneEvent, runConstructor, sendKeysElem, waitUntil } from '@brightspace-ui/testing';
```